### ISSUE TITLE
Actually remove followers if they are deleted remotely

### ISSUE THREAD

**Issue by gabek** [owncast/owncast#2200]
### Share your bug report, feature request, or comment.

We are currently not handling remote `Delete` ActivityPub activities and we should be. Since our implementation of ActivityPub doesn't deal with threads and account profiles, just the "Followers" it's not the end of the world, but it does end up with an incorrect list of followers if one of them deletes their account and we don't reflect that.

**Comment by brokenintuition** [comment]
I'd be interested in working on this one

**Comment by gabek** [comment]
@brokenintuition Awesome! While I don't want to discourage anybody from working on anything, this issue will require digging a bit deeper into the Go codebase of Owncast and working on the ActivityPub and database layers. If you're cool with that, it's all yours!

**Comment by brokenintuition** [comment]
Yeah I've been looking to work on some bigger issues and learn more about ActivityPub so I'm all aboard

**Comment by gabek** [comment]
Wohoo! Feel free to drop by our [chat](https://owncast.rocket.chat/channel/hacktoberfest) if you'd like to discuss in real time at all.

**Comment by gabek** [comment]
Followers are now cleaned up proactively instead of just reactively, so I'm going to close this.
